### PR TITLE
Fix case with different classes having fields with the same names

### PR DESCRIPTION
**Pull request by geralt-encore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,6 +189,7 @@ public class Tweet {
 ```
 
 -`Kotlin`:
```
Comment (nikitin-da) [inline README.md]: excess dash

**Comment by geralt-encore** [comment]
@artem-zinnatullin @nikitin-da how about making a release with Kotlin related changes? Seems reasonable considering the fact of its growing popularity.

**Comment by nikitin-da** [comment]
@geralt-encore I agree that there is huge pack of changes and Kotlin changes required to release ASAP, but it seems RxJava2 support will break compatibility again. 
It will lead to two major releases at once =(

**Comment by geralt-encore** [comment]
I see your point, but I have to concerns regarding it. First, it is not really a breaking API change - it is fixing something that was initially broken, so I think that the fact that something doesn't work in a broken way, which it was never supposed to work is a good thing. Second, we have zero ideas about the timeline of RxJava 2 support =(

**Comment by nikitin-da** [comment]
Unfortunately we have already merged [interceptors](https://github.com/pushtorefresh/storio/pull/771) and we should roll it back from master or release it with major version(
@artem-zinnatullin will it be ok to release fast v2 and v3?

**Comment by geralt-encore** [comment]
Ouch, I completely forgot about interceptors change... Then, yeah, you are totally right.

**Comment by artem-zinnatullin** [comment]
@nikitin-da yes, I think we can release v2 and then v3, that's what semantic versioning is for :)

**Comment by nikitin-da** [comment]
Ok, let's release v2

**Comment by artem-zinnatullin** [comment]
See #806

**Pull request merged**